rootfs/templates/kops/default.yaml

```diff
@@ -18,37 +18,45 @@ spec:
   api:
     loadBalancer:
       type: Public
+  {{- if bool (getenv "KOPS_AUTHORIZATION_RBAC_ENABLED") }}
+  kubeAPIServer:
```

Comment:
Do we need `authorizationMode`?

Reply:
I don't think so, because it doesn't specified in cluster.spec

Reply:
Bump

Reply:
i read about `kubeAPIServer` props, so the only thing that could be usefull is `authorizationMode: RBAC,AlwaysAllow`. It explicitly declares authorization modes.